string sensingElement              #传感器名称
bool dataType                      #数据类型。0：bool；  1：数组
bool data                          #状态
float64[4] array                   #状态